README.md

```diff
@@ -1,2 +1,54 @@
-# rtx-kg2-gateway
+# RTX-KG2 Gateway
+
 Enabling RTX-KG2 data access through various means.
+
+## Installation
+
+Please use Python [`poetry`](https://python-poetry.org/) to run and install related content.
+The Poetry environment for this project includes dependencies which help run IDE environments, manage the data, and run workflows.
+
+```bash
+# after installing poetry, create the environment
+poetry install
+```
+
+## Development
+
+### Jupyter Lab
+
+Please follow installation steps above and then use a relevant Jupyter environment to open and explore the notebooks under the `notebooks` directory.
+
+```bash
+# after creating poetry environment, run jupyter
+poetry run jupyter lab
+```
+
+### Poe the Poet
```

Comment:
This section looks like it's about running tasks to generate the data, so I'd name it something like "Generating the Data" rather than the specific tool you're using to launch the tasks that does that (i.e., "Poe the Poet").

Reply:
Great point! I'll get this updated.